CharlieBackend.Business/Services/ScheduleServiceFolder/ScheduleService.cs

```diff
@@ -189,6 +190,32 @@ public async Task<Result<ScheduledEventDTO>> UpdateScheduledEventByID(long sched
             return Result<ScheduledEventDTO>.GetSuccess(_mapper.Map<ScheduledEventDTO>(item));
         }
 
+        public async Task<Result<ScheduledEventDTO>> AddSingleScheduledEvent(CreateScheduledEventDto createSingleScheduleRequest)
```

Comment:
Please create event through Event controller and pass only required data. You don't need Range and Context here. You need only several fields from the Context as GroupId and MentorId.
Also please add unit tests to newly created methods later.

Reply:
Create if needed new DTO model for adding an event, add fluent validation for this model.